cli/tsc/99_main_compiler.js

```diff
@@ -100,6 +100,8 @@ delete Object.prototype.__proto__;
     "PermissionStatus",
     "hostname",
     "ppid",
+    "HttpClient",
+    "createHttpClient",
```

Comment:
Actually these should not be in the compiler... We don't use fetch in the compiler snapshot.

Reply:
This is the list of Deno global properties where we need to replace the error message from `Property 'loadavg' does not exist on type 'typeof Deno'.` to  `Property 'loadavg' does not exist on type 'typeof Deno'. 'Deno.loadavg' is an unstable API. Did you forget to run with the '--unstable' flag?` if you don't have unstable enabled. This needs to be in the compiler.